ports/atmel-samd/boards/hallowing_m0_express/board.c

```diff
@@ -93,6 +93,8 @@ void board_init(void) {
         16, // Color depth
         false, // Grayscale
         false, // Pixels in a byte share a row. Only used for depth < 8
+        1, // bytes per cell. Only valid for depths < 8
```

Comment:
At some point maybe this should turn into a struct, which you could initialize with slot names, and then pass a pointer to the struct, to save passing so many positional arguments. Maybe it could be static. Doesn't have to be now.

Reply:
Ya, wish C had kwargs. Does using a struct prevent the compiler from optimizing things?